### ISSUE TITLE
help(altair) raises ValueError (in last Python 3.5 compatible release)

### ISSUE THREAD

**Issue by jonathangjertsen** [altair-viz/altair#2384]
I didn't see this mentioned in the changelog, but trying to call `help(...)` on altair module using the last Python 3.5 compatible release results in a ValueError.

I've tested in a different environment (Python 3.9, Altair 4.1.0) - there is no issue there.

I know 3.5 is unsupported so free to close the issue based on that, just wondering whether this is a known bug with a workaround.

```
>python
Python 3.5.6 |Anaconda, Inc.| (default, Aug 26 2018, 16:05:27) [MSC v.1900 64 bit (AMD64)] on win32
Type "help", "copyright", "credits" or "license" for more information.
>>> import altair
>>> help(altair)
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "[redacted]\lib\_sitebuiltins.py", line 103, in __call__
    return pydoc.help(*args, **kwds)
  File "[redacted]\lib\pydoc.py", line 1851, in __call__
    self.help(request)
  File "[redacted]\lib\pydoc.py", line 1904, in help
    else: doc(request, 'Help on %s:', output=self._output)
  File "[redacted]\lib\pydoc.py", line 1638, in doc
    pager(render_doc(thing, title, forceload))
  File "[redacted]\lib\pydoc.py", line 1631, in render_doc
    return title % desc + '\n\n' + renderer.document(object, name)
  File "[redacted]\lib\pydoc.py", line 375, in document
    if inspect.ismodule(object): return self.docmodule(*args)
  File "[redacted]\lib\pydoc.py", line 1172, in docmodule
    contents.append(self.docother(value, key, name, maxlen=70))
  File "[redacted]\lib\pydoc.py", line 1395, in docother
    repr = self.repr(object)
  File "[redacted]\lib\reprlib.py", line 55, in repr
    return self.repr1(x, self.maxlevel)
  File "[redacted]\lib\pydoc.py", line 1031, in repr1
    return cram(stripid(repr(x)), self.maxother)
  File "[redacted]\lib\site-packages\altair\expr\core.py", line 7, in __repr__
    raise ValueError("must use datum.attribute")
ValueError: must use datum.attribute
```

**Comment by jakevdp** [comment]
Which version of Altair did you use?

**Comment by jonathangjertsen** [comment]
4.0.1

**Comment by jakevdp** [comment]
This bug was fixed in #1974, which is part of the 4.1 release.

**Comment by jonathangjertsen** [comment]
Thanks!

I assume there are no plans to continue supporting Python 3.5. so the following workaround will have to do for those of us working with a code base that isn't upgraded yet:

```Python
import altair
altair.expr.core.DatumType.__repr__ = lambda self: "datum"
help(altair) # works
```
